src/components/AddRepoForm.js

```diff
@@ -18,9 +19,17 @@ function AddRepoForm({goalsId, onGoalAdded}) {
     }
 
     const [isValid, repoUrl] = isValidRepoUrl(urlRef.current.value.replace(/\s+/g, ""));
+    const statusCode = await repoStatusCode(repoUrl);
+
     if (!isValid) {
       urlRef.current.focus();
-      console.warn("Invalid GitHub repository!");
+      alert("Invalid GitHub repository!");
```

Comment:
Instead of an alert, can you use the ErrorMessage, it is done in the Repository. 

https://github.com/open-sauced/open-sauced/blob/master/src/components/Repository.js#L67

Reply:
@bdougie Done! https://www.loom.com/share/668096d3e518401e8c06340da0e3b5bb